sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -56,21 +57,40 @@ internal static List<TelemetryItem> OtelToAzureMonitorLogs(Batch<LogRecord> batc
             {
                 try
                 {
-                    telemetryItem = new TelemetryItem(logRecord, resource, instrumentationKey);
-                    if (logRecord.Exception != null)
+                    var properties = new ChangeTrackingDictionary<string, string>();
+                    ProcessLogRecordProperties(logRecord, properties, out string? message, out string? eventName);
+
+                    if (message is not null && logRecord.Exception is not null)
```

Comment:
If logRecord is exception we don't need to have additional check on CustomEventAttributeName and could use the existing implementation.

Reply:
I did a large refactor of my LogsHelper changes since this comment. I believe this is no longer relevant.